pkg/humio/client.go

```diff
@@ -22,15 +20,8 @@ type Client struct {
 }
 
 type Config struct {
-	Address            *url.URL
-	Token              string
-	InsecureSkipVerify bool
-	TlsClientAuth      bool
-	TlsAuthWithCACert  bool
-	TlsCACert          string
-	TlsClientCert      string
-	TlsClientKey       string
-	TlsServerName      string
```

Comment:
Do we need to backport these fields? Do old data source still work?

Reply:
They should still work as in the frontend the settings were still controlled via the `DefaultHTTPSettings` component so the default http client in the backend should pick them up fine.